main.py

```diff
@@ -1,10 +1,39 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as player_json:
+        players = json.load(player_json)
+        for player in players:
+            race = Race.objects.get_or_create(
```

Comment:
The get_or_create method returns a tuple (object, created). Ensure you are handling both elements of this tuple to avoid confusion and potential errors.